packages/video_player/lib/video_player.dart

```diff
@@ -305,16 +383,23 @@ class VideoPlayerController extends ValueNotifier<VideoPlayerValue> {
     super.dispose();
   }
 
+  /// Starts playing the video.
+  ///
+  /// This finishes as soon as the "play" command has been sent to the platform,
+  /// not when playback is complete.
```

Comment:
maybe: `This method returns as soon as the playback has started, not when it is complete.`

Reply:
This is referring specifically to the future. It completes once `"play"` itself starts, not when playback is complete. I thought that was potentially confusing so I mentioned it. I'll clear this up to say "The future that is returned from this method finishes as soon as..."